src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -94,6 +118,9 @@ export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: an
 	if (typeof displayValue === 'number') {
 		displayValue = displayValue.toString();
 	}
+	if (displayValue instanceof Array) {
+		displayValue = displayValue.toString();
```

Comment:
```
let arr = ['1', 123, 'hello']
undefined
arr.toString()
"1,123,hello"
```

is this intended?

Reply:
yes. I have a column with array as a data, so I need to display it. SlickGrid in core has this functionality.